repos/fdbt-site/src/pages/selectCaps.tsx

```diff
@@ -0,0 +1,167 @@
+                                                <CapsCard key={item.cap.name} caps={item} selectedId={selectedId} />
+                                            ))
+                                        ) : (
+                                            <p className="govuk-body govuk-error-message">
+                                                <a href="/viewCaps">Create a cap in operator settings.</a>
+                                            </p>
+                                        )}
+                                    </div>
+                                </div>
+
+                                <div className="govuk-radios__item">
+                                    <input
+                                        className="govuk-radios__input"
+                                        id="valid-days-not-required"
+                                        name="capChoice"
+                                        type="radio"
+                                        value="no"
+                                        data-aria-controls="conditional-caps-2"
+                                        defaultChecked={!selectedId}
+                                    />
+                                    <label className="govuk-label govuk-radios__label" htmlFor="no-choice">
+                                        No
+                                    </label>
+                                </div>
+                            </div>
+                        </fieldset>
+                    </div>
+
+                    <input
+                        type="submit"
+                        value="Continue"
+                        id="continue-button"
+                        className="govuk-button govuk-!-margin-right-2"
+                    />
+
+                    <a className="govuk-button govuk-button--secondary" href="/viewCaps">
+                        Create new
+                    </a>
+                </>
+            </CsrfForm>
+        </FullColumnLayout>
+    );
+};
+
+const CapsCard = ({ caps, selectedId }: { caps: CapInfo; selectedId: number | null }): ReactElement => {
+    return (
+        <div className="card">
+            <div className="card__body caps">
+                <div className="govuk-radios">
+                    <div className="govuk-radios__item card__selector">
+                        <input
+                            className="govuk-radios__input"
+                            id={`${caps.cap.name}-radio`}
+                            name="cap"
+                            type="radio"
+                            value={caps.id}
+                            aria-label={caps.cap.name}
+                            defaultChecked={selectedId === caps.id}
+                        />
+                        {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+                        <label className="govuk-label govuk-radios__label" />
```

Comment:
Do we need this label? what is it doing?

Reply:
/Users/ajaffer1/Desktop/my-app/create-data/repos/fdbt-site/src/pages/selectCaps.tsx
site:   136:25  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control

Reply:
Is that the error that comes up if you remove the label entirely?

Reply:
@alainajaffer ?